src/components/portrait/portrait.pw.tsx

```diff
@@ -0,0 +1,486 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import { getDataElementByValue } from "../../../playwright/components";
+import {
+  portraitImage,
+  portraitInitials,
+  portraitPreview,
+} from "../../../playwright/components/portrait/index";
+
+import { icon } from "../../../playwright/components/index";
+
+import {
+  CHARACTERS,
+  COLOR,
+  SIZE,
+  VALIDATION,
+} from "../../../playwright/support/constants";
+import { checkAccessibility } from "../../../playwright/support/helper";
+import Box from "../../../src/components/box";
+import {
+  PORTRAIT_SIZE_PARAMS,
+  PORTRAIT_SIZES,
+} from "../../../src/components/portrait/portrait.config";
+import {
+  PortraitComponent,
+  PortraitDefaultComponent,
+} from "./components.test-pw";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const colors = [
+  ["orange", COLOR.ORANGE],
+  ["red", COLOR.RED],
+  ["black", COLOR.BLACK],
+  ["brown", COLOR.BROWN],
+];
+const testImage =
+  "https://www.gravatar.com/avatar/cceff1ad774bf89b1f75cb6e5005333c?s=40&d=404";
+const imageURL = [
+  "https://avataaars.io/?avatarStyle=Transparent&topType=LongHairStraight&accessoriesType=Blank&hairColor=BrownDark&facialHairType=Blank&clotheType=BlazerShirt&eyeType=Default&eyebrowType=Default&mouthType=Default&skinColor=Light",
+  "https://www.gravatar.com/avatar/05c1c705ee45d7ae88b80b3a8866ddaa?s=24&d=404",
+];
+
+const portraitSizes = PORTRAIT_SIZES.map((size) => [
+  size,
+  PORTRAIT_SIZE_PARAMS[size].dimensions,
+]);
+
+const visibilityValues: [boolean, string][] = [
+  [true, "with"],
+  [false, "without"],
+];
+
+test.describe("Portrait component", () => {
```

Comment:
```suggestion
test.describe("check props for Portrait component", () => {
```

**suggestion** (non-blocking): This is how we've described this in other components, and also how it was done in Cypress. I don't think we can just use "Portrait component" because this isn't the only `test.describe` block, and all of them are obviously testing the portrait component.